client/src/pages/Home.svelte

```diff
@@ -49,7 +51,8 @@
         text-align: center;
         padding: 5pt;
         border: 1pt;

```

Comment:
In a future PR, we should reconsider these hard-coded values. We may want to use `rem` units instead so that the design is responsive.